app/scripts/modules/core/application/modal/editApplication.html

```diff
@@ -164,6 +164,16 @@ <h3 data-purpose="modal-header">Edit Application</h3>
         </div>
       </div>
 
+      <div class="form-group row">
+        <div class="col-sm-3 sm-label-right">Pipeline Behavior</div>
+        <div class="col-sm-9">
+          <input type="checkbox"
+                 ng-model="editApp.applicationAttributes.enableRestartRunningExecutions">
+          Enable restarting running pipeline executions
```

Comment:
+1 - `Allow restarting running pipelines` is probably better wording in both places.